### ISSUE TITLE
Consider renaming the `@prisma/sdk` package to reduce confusion

### ISSUE THREAD

**Issue by nikolasburk** [prisma/prisma#10725]
The name `@prisma/sdk` is a bit confusing since it's not really an SDK, so we might consider renaming it. Based on this internal [Slack convo](https://prisma-company.slack.com/archives/CFM8D5VHC/p1638957631137800):

![image](https://user-images.githubusercontent.com/4058327/146371871-9be7f091-9e9f-461a-8184-a0d7b3ee8e31.png)

**Comment by YassinEldeeb** [comment]
Yeah it should be renamed to be like "common" or "helpers" cause a lot of people think that it's an API for external usage from its name, but it's a really good point that this is a breaking change cause there are a lot of repositories depending on `@prisma/sdk`.

**Comment by Jolg42** [comment]
Thanks @YassinEldeeb for your feedback!

Note: We are considering adding a deprecation message soon and rename it for Prisma 4 (date undefined yet)

**Comment by Jolg42** [comment]
[Internal]

Me, Pierre and Alexey created a draft of a plan the renaming
https://www.notion.so/prismaio/Renaming-prisma-sdk-and-cleaning-up-internal-packages-f217f9ba96aa40fd91f47995b9266dc3
Work in progress, nothing planned yet, but this would be the general direction we want to go for.

**Comment by jkomyno** [comment]
Can we move this issue to "done"?

**Comment by Jolg42** [comment]
The internal package `@prisma/sdk` will be available under the new, more explicit name `@prisma/internals` from Prisma version `4.0.0`.

`@prisma/sdk` will be published for Prisma `4.0.0` and marked as deprecated on npm, it will be the last version published in its current shape.
The upgrade path would be to rename the usage `@prisma/sdk` to `@prisma/internals`. 

⚠️ Note that functionality of this internal package can be changed or even removed any time and our SemVer guarantees (no breaking changes without major version increase) are not valid here.

Making the intended use of that package explicit (internal only) is important as it helps to manage expectations with users.

This is technically not a breaking change as usage of the `sdk` package was not documented nor supported.

If you are using this package, it would be helpful if you could help us gain an understanding of where, how and why you are using it. Your feedback will be valuable to us to define a better API. Please share this information at [https://github.com/prisma/prisma/discussions/13877](https://github.com/prisma/prisma/discussions/13877) - Thanks!

npm packages
https://www.npmjs.com/package/@prisma/sdk
https://www.npmjs.com/package/@prisma/internals
